terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/AbstractReloadableCodeListTest.java

```diff
@@ -15,24 +15,48 @@
  */
 package org.terasoluna.gfw.common.codelist;
 
-import static org.junit.Assert.assertThat;
 import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.terasoluna.gfw.common.codelist.AbstractReloadableCodeList;
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.Context;
 
 /**
  * Abstract class for the reloadable codelist functionality
  */
 public class AbstractReloadableCodeListTest {
 
+    private final String LOGBACK_UNIT_TEST_FILE_PATH = "src/test/resources/logback-unittest.xml";
+
+    private final String LOGBACK_DEFAULT_FILE_PATH = "src/test/resources/logback.xml";
+
+    private String logbackUnitTestFilePath = LOGBACK_UNIT_TEST_FILE_PATH;
+
+    private String logbackDefaultFilePath = LOGBACK_DEFAULT_FILE_PATH;
+
+    private Logger logger;
+
+    private Context context;
+
+    private JoranConfigurator configurator;

```

Comment:
https://github.com/terasolunaorg/terasoluna-gfw/commit/053a4b3ff2cd650dcfaabed740e877b5708fc8a5#diff-98d97d117558481f40a4c226e189b525R31
I not intended `before` method but `@Before` method.
If you want to call method changing logback settings, you should make utility class. Because test class has no semantic inheritance relationship to abstract `ChangingLogbackFile` class.